[BE8]정주신

ehrl1225 opened the pull request.

ehrl1225 committed:
init

ehrl1225 committed:
simplified wise saying show controller

ehrl1225 committed:
필요 없는 파일 삭제

Kimgooner commented on src/main/java/com/back/step15/repository/MySqlManager.java:
```diff
@@ -0,0 +1,280 @@
+    Connection conn;
+
+
+    private String getInsertSql(String content, String author) {
+        StringBuilder sql = new StringBuilder();
+        sql.append("insert into WiseSaying(content, author) values(\'");
+        sql.append(content);
+        sql.append("\', \'");
+        sql.append(author);
+        sql.append("\');");
+        return sql.toString();
+    }
```
해당 메서드처럼 SQL 쿼리를 직접 만드는 것 보단, PreparedStatement를 사용하여 인자 값을 전달하는 방식이 더 괜찮아 보입니다.

```java
String sql = "INSERT INTO WiseSaying(content, author) VALUES(?, ?)";
PreparedStatement pstmt = conn.prepareStatement(sql);
pstmt.setString(1, content); // 첫번째 ?에 인자값 전달 (=content)
pstmt.setString(2, author); // 두번째 ?에 인자값 전달 (=author)
```

ehrl1225 replied:
네 그러려고요.

ehrl1225 committed:
일부 수정

ehrl1225 committed:
처음부터 다시 작성함
SimpleDb 연동시킴
연동시키면서 SimpleDbTest는 실패가 뜨지만
WiseSayingControllerTest는 정상 작동이라 올림
